lua/tardis/cl_options.lua

```diff
@@ -17,7 +17,7 @@ function TARDIS:ChangeOption(id,data)
 	text:SetPos(10,30)
 	text:SetTextColor(color_white)
 	
-	local value=TARDIS:GetSetting(id,data.value)
+	local value=TARDIS:GetSetting(id,data.value,LocalPlayer())
```

Comment:
What's the reason behind this change?

Reply:
Settings added both in the shared context wouldn't properly network on the GUI for some reason (would look like they hadn't saved), however they were still being set and unset properly when checked from code